pages/dkp/kommander/2.2/cli/dkp/create/cluster/aks/index.md

```diff
@@ -3,7 +3,7 @@ layout: layout.pug
 navigationTitle:  dkp create cluster aks
 title:  dkp create cluster aks
 menuWeight: 10
-excerpt: Create a Konvoy cluster in AKS
+excerpt: Create a DKP cluster in AKS
 notes: Automatically generated, DO NOT EDIT
```

Comment:
I think this will be overwritten since it's automatically generated.

Reply:
How is that generated? How should I update it? Thanks

Reply:
@floridoo where would these commands need to be updated?

Reply:
Looks like all the CLI changes in this PR belong to Konvoy, so they would need to be changed in the [Konvoy 2 repo](https://github.com/mesosphere/konvoy2), under `cmd/konvoy/app`.
E.g. for this change, this is the right place:
https://github.com/mesosphere/konvoy2/blob/d3a75e78ebb8dd4a1aa754d955324fef216a56e0/cmd/konvoy/app/cluster/create_capz.go#L81